libs/cassandra-util/test/integration/Operations/Identity.hs

```diff
@@ -0,0 +1,25 @@
+module Operations.Identity where
+
+import Imports
+import Data.Text (Text)
+import Data.Functor.Identity (Identity)
+import Test.Tasty
+import Test.Tasty.HUnit
+import Cassandra (QueryParams(QueryParams), QueryString, R, Consistency(One), runClient, query)
+
+import qualified Cassandra as DB
+
+tests :: DB.ClientState -> IO TestTree
+tests db = do
+  return $ testGroup "Identity" [
+    testCase "Identity works" $ do
+        out <- testIdentity db
+        assertEqual "CQL version match." "Identity 3.4.4" (show out)
+        --test @? "CQL version mismatch"
+    ]
+
+testIdentity :: DB.ClientState -> IO [Identity Text]
+testIdentity db = do
+  let q = "SELECT cql_version from system.local" :: QueryString R () (Identity Text)
+  let p = QueryParams One False () Nothing Nothing Nothing Nothing
+  runClient db $ query q p
```

Comment:
I know we have lots of single-char variables all over the place, but this is something I would like to outlaw.  `qry` and `prms`?  or `iquery` and `iparams`?